test/methods/build-function.test.ts

```diff
@@ -1,13 +1,15 @@
-import { describe, it, expect } from 'vitest';
-import { buildFunction } from '../../src/methods/build-function.ts';
+import { describe, it, expect } from "vitest";
```

Comment:
We should add a `prettier` config in this repo so that styling is standardized.

Reply:
Sounds good. I'll do this in a separate PR